packages/amazonq/src/codeActions.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+export class CodeActions implements vscode.CodeActionProvider {
```

Comment:
Notice that there is a [app/chat/](https://github.com/aws/aws-toolkit-vscode/tree/cb6bd3006eeeaba061c51b6a30e032232402b6c1/packages/amazonq/src/app/chat) for application code for AmazonQ. 

If this CodeActions module is generally useful for all Q features, it could live in `packages/amazonq/src/util/` or `packages/amazonq/src/shared/`. If it's for a particular Q feature, it should live in `app/<feature>/`.

Or it could live in `app/` directly. 

The main point is that the top-level `src/` directory isn't the right place for most code files.

Reply:
Good call. Will review